src/StatsGenerator.ts

```diff
@@ -25,6 +25,13 @@ export class StatsGenerator {
         return this.lastResult;
     }
 
+    /**
+     * Returns last update time
+     */
+    public getLastUpdate(){
+        return this.lastResult.timestamp;
+    }
+
```

Comment:
Is this needed?  Could do `getLastResult().timestamp` instead

Reply:
Can do this